### DISCUSSION TITLE
Unable to enter username

### DISCUSSION THREAD

**Discussion by biasedagostino** [globaleaks/globaleaks-whistleblowing-software#4164] [Q&A]
Hello everyone.
In an instance of one of our customers when I try to enter from the login page I am NOT asked for the username and therefore I cannot log in because my password is not recognized.
I tried to update the version via script but then it would not start due to TOR problems. Then I uninstalled TOR and relaunched the update script.
Now TOR is back to working but the login page is still without a username.
Checking the request it would seem that a username is already prepared in the POST (in the format that I would associate with an ID or UUID). I can't solve this problem, can someone help me?
Globaleaks version 5.0.0

![Screenshot 2024-08-22 151432](https://github.com/user-attachments/assets/28c55efb-bdca-4f00-8c87-2b98b3e8617a)

**Comment by evilaliv3** [comment]
Hello @biasedagostino 

I guess you may have enabled by mistake the option "Login semplicicato" present in the advanced settings and that on that system exists on that system exist only one recipient user. In this configuration when you provide a password the system attempts a login with that user.
If you need to access as administrator you may login via /#/admin

If you could provide me the url of the system here or via community.globaleaks.org i may give it a look and try to provide assistance.

**Comment by biasedagostino** [comment]
Ciao @evilaliv3 ,
possiamo continuare in italiano, ti ringrazio per il tuo consiglio ma putroppo non posso verificare poichè anche attraverso la url /#/admin mi ritrovo nella stessa situazione.

Certo, posso inviarti la url, c'è qualche modo in cui posso inviartela privatamente o in chiaro qui?

Grazie mille

**Comment by biasedagostino** [comment]
Ne approfitto per segnalarti che me ne sono accorto in seguito ad una mail con una segnalazione di errore :
`Handler [StaticFileHandler] exceeded execution threshold (of 120 secs) with an execution time of 134.00 seconds
`
Grazie

**Reply by evilaliv3** [reply]
Privatamente se vuoi mi trovi sulla nostra chat di community community.globaleaks.org con lo username evilaliv3

**Comment by biasedagostino** [comment]
Grazie per il supporto @evilaliv3. Per chi si trovasse su questa pagina con lo stesso problema (relativo alla verrsione 5.0.0) per accedere al login amministrativo dovrà andare alla url : "NOMESITO/#/admin/". A questo punto potrà accedere con user e password. Il mio consiglio (seguendo il suggerimento di @evilaliv3 ) è disabilitare il login semplificato andando in IMPOSTAZIONI -> AVANZATE -> disabilita login semplificato.
